lib/files_finder.ex

```diff
@@ -10,9 +10,12 @@ defmodule ExcellentMigrations.FilesFinder do
   end
 
   def relevant_file?(path, start_after) do
+    migrations_paths =
+      Application.get_env(:excellent_migrations, :migrations_paths, ["migrations/"])
```

Comment:
I like that a list of user defined paths is passed here (not just a single one) 👍🏻